marlowe-cli/src/Language/Marlowe/CLI/Test/Types.hs

```diff
@@ -125,10 +128,10 @@ data ScriptTest =
   ScriptTest
   {
     stTestName         :: String             -- ^ The name of the test.
-  , stSlotLength       :: Integer            -- ^ The slot length, in milliseconds.
-  , stSlotZeroOffset   :: Integer            -- ^ The effective POSIX time of slot zero, in milliseconds.
-  , stInitialContract  :: Contract           -- ^ The contract.
-  , stInitialState     :: State              -- ^ The the contract's initial state.
+  -- , stSlotLength       :: Integer            -- ^ The slot length, in milliseconds.
+  -- , stSlotZeroOffset   :: Integer            -- ^ The effective POSIX time of slot zero, in milliseconds.
+  -- , stInitialContract  :: Contract           -- ^ The contract.
+  -- , stInitialState     :: State              -- ^ The the contract's initial state.
```

Comment:
@bwbush it seems that we somehow missed your intention Brian and make the `Initialize` operation a contract carrier and we introduced a state for multiple contracts.

Do you think that we should simplify that and assume a single contract per test?

Do you think that this can be valuable extension and we can leave it as it is implemented now?